src/functions/jitTippy.js

```diff
@@ -1,7 +1,11 @@
 import tippy from "tippy.js";
+import { hideAll } from "tippy.js";
 import getSuttaBlurb from "../functionsBuilding/getSuttaBlurb.js";
 
-const params = { delay: [500, null], touch: ["hold", 500], allowHTML: true, interactive: true };
+const MOUSEOVERDELAY = 350;
+const TOUCHDELAY = 250;
```

Comment:
Now these are pulled out into convenient, meaningful constants (which will update the value in both places)